pulp-in-one-container.md

```diff
@@ -23,9 +23,15 @@ $ mkdir settings pulp_storage pgsql containers
 $ echo "CONTENT_ORIGIN='http://$(hostname):8080'
 ANSIBLE_API_HOSTNAME='http://$(hostname):8080'
 ANSIBLE_CONTENT_HOSTNAME='http://$(hostname):8080/pulp/content'
+TOKEN_SERVER = "http://(hostname):24817/token/"
+TOKEN_SIGNATURE_ALGORITHM = 'ES256'
+PUBLIC_KEY_PATH = '/tmp/public_key.pem'
+PRIVATE_KEY_PATH = '/tmp/private_key.pem'
 CACHE_ENABLED=True" >> settings/settings.py
 ```
 
+Refer to plugin settings configuration docs for more.
```

Comment:
Perhaps, we can be more specific here and reference the pulp_container documentation. I do not know if there are any other plugins that require the customization to the settings file before booting up the container.

Reply:
https://github.com/pulp/pulp_container/issues/1550

Reply:
That's why it is better to stay generic because you don't know, plus things can change in future and you will need to otherwise keep track of each plugin settings.